src/referee/referee/referee_node.py

```diff
@@ -0,0 +1,77 @@
+import socket
+import struct
+import rclpy
+import sys
+import os
+from rclpy.node import Node
+from std_msgs.msg import String
+from google.protobuf import json_format
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), 'proto')))
+
+try:
+    import ssl_gc_referee_message_pb2
+except ImportError as e:
+    print(f"Error importing Protobuf: {e}")
+    sys.exit(1)
+
+MULTICAST_GROUP = '224.5.23.1'
+MULTICAST_PORT = 10003
+BUFFER_SIZE = 1024
+
+class MulticastListener(Node):
+    def __init__(self):
+        super().__init__('multicast_listener')
+        self.publisher_ = self.create_publisher(String, 'referee_messages', 10)
+        self.last_message = None  # Variável para armazenar a última mensagem
+
+        # Configuração do socket multicast
```

Comment:
There is a client class doing the socket configuration, but not being used here.